js/personal.js

```diff
@@ -80,6 +80,7 @@ View.prototype = {
         var html = Handlebars.templates['personal']({
             loaded: this._persons.isLoaded(),
             persons: this._persons.getAll(),
+            appPath: OC.generateUrl('/'),
```

Comment:
Hi @feedthedogs 
First of all, thank you very much for helping us !!. :smile: 

This was something that was pending .. :wink: 

Here you can generate two paths to avoid using '../../' :confused: in the template..

> appPath: OC.generateUrl('/apps/facerecognition') /* To thumbnails*/

.. and

> filePath: OC.generateUrl('/f') /* To open on file links */

Reply:
I tried what you suggested, but it didn't work.
The reason for the ../../ is the handlebar's variables are within the foreach sections
another option if you don't like that notation I think you can use root